### PR TITLE
added basic no-flash

### PR DESCRIPTION
**Pull request by veb**

### CODE REVIEW

File: Wall/main.cpp [hunk 1]
```diff
@@ -271,8 +271,13 @@ int testLocalPlayerAddress(uint64_t clientBase) {
     // Developer Comment:
     // int health              = mem->read<int>(playerAddress + 0x134);
     // We don't read this because it was never been used
+    int flashed             = mem->read<int>(playerAddress + 0xABF8);
```
Comment (aKalisch) [inline Wall/main.cpp:274]: @veb You read the value as integer

File: Wall/main.cpp [hunk 2]
```diff
@@ -271,8 +271,13 @@ int testLocalPlayerAddress(uint64_t clientBase) {
...
     int iTeamNum            = mem->read<int>(playerAddress + 0x128);
 
+    if(flashed) {
```
Comment (aKalisch) [inline Wall/main.cpp:277]: @veb here you check for a boolean

File: Wall/main.cpp [hunk 3]
```diff
@@ -271,8 +271,13 @@ int testLocalPlayerAddress(uint64_t clientBase) {
...
+        mem->write<float>(playerAddress + 0xABF8, 0);
```
Comment (aKalisch) [inline Wall/main.cpp:278]: @veb and here you write a float value

File: Wall/main.cpp [hunk 4]
```diff
@@ -271,8 +271,13 @@ int testLocalPlayerAddress(uint64_t clientBase) {
...
+    }
+    
     return iTeamNum;
 }
```
Comment (aKalisch) [inline Wall/main.cpp:283]: This seams to be inconsistant. PR will not be merged by now. Also this is csgoGlow and not a "Multi-Hack" :)

**Comment by veb** [comment]
Ha. Thanks for the pointers. ;)

**Pull request closed without merging**